game/static/game/css/forum.css

```
@@ -935,4 +1013,104 @@ html {
+    font-weight: 700;
+    color: #ffffff;
+}
+
+.upvote-count  { color: #6ddb8b; }
+.downvote-count { color: #e06c75; }
+
+.reply-vote-btn {
+    width: 30px;
+    height: 30px;
+
+    background: transparent;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)